src/myfunc.cc

```diff
@@ -8,7 +8,8 @@
 void
 myfunc3(void)
 {
```

Comment:
[modernize-redundant-void-arg]: redundant void argument list in function definition